modular_zubbers/modules/lpa_teshari/_DEFINES/species_defines.dm

```diff
@@ -0,0 +1,9 @@
+#define RAPTOR_TEMP_OFFSET -30
+#define RAPTOR_HEATMOD 1.1
+#define RAPTOR_COLDMOD 0.8
+
+#define RAPTOR_HUNGERMOD 0.5
+
+#define TESHARI_ALT_TEMP_OFFSET -50
+#define TESHARI_ALT_HEATMOD 1.4
+#define TESHARI_ALT_COLDMOD 0.2
```

Comment:
This needs undefines at the end of the folder.

Reply:
These defines are used in another file, I can't remove them at the end of the this file.

Reply:
You can just go to the last file in the folder and undefine it there, cuts on compiler cache sizes